presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -181,10 +181,11 @@ public void testFailAcidBeforeHive3()
 
     private static String hiveTableProperties(TransactionalTableType transactionalTableType, BucketingType bucketingType)
     {
-        return Stream.concat(
-                transactionalTableType.getHiveTableProperties().stream(),
-                bucketingType.getHiveTableProperties().stream())
```

Comment:
you did not like the Stream.concat way?

Reply:
It does not support 3 args

Reply:
Stream.of(stream1, ...).flatMap(Function.identity())